sormas-backend/src/main/java/de/symeda/sormas/backend/person/PersonFacadeEjb.java

```diff
@@ -1350,10 +1346,7 @@ public List<PersonIndexDto> getIndexList(PersonCriteria criteria, Integer first,
 			person.get(Person.CHANGE_DATE),
 			JurisdictionHelper.booleanSelector(cb, service.inJurisdictionOrOwned(personQueryContext)));
 
-		Predicate filter = createIndexListFilter(criteria, personQueryContext);
-		if (filter != null) {
-			cq.where(filter);
-		}
+		cq.where(person.get(Person.ID).in(indexListIds));
```

Comment:
The in clause could surpass the parameter limit, so please encapsulate the data fetch query with `IterableHelper.executeBatched` It also deals with empty id list so no additional empty check is then needed.